cerberus/validator.py

```diff
@@ -1065,11 +1085,12 @@ def _validate_minlength(self, min_length, field, value):
     def _validate_nullable(self, nullable, field, value):
         """ {'type': 'boolean'} """
         if value is None:
-            if nullable:
-                return True
-            else:
+            if not nullable:
                 self._error(field, errors.NOT_NULLABLE)
-                return True
+            self._drop_remaining_rules(
```

Comment:
Maybe we should remove all the rules beside dependencies? The current behavior might breaks validator with custom rules since the rules won't be removed from the queue.